### PR TITLE
Feat/url helper

### PR DESCRIPTION
**Pull request by chaosmonster**

### CODE REVIEW

File: src/renderer.ts [hunk 1]
```diff
@@ -7,14 +7,19 @@ import {TemplateData} from './template-data.interface';
 import YAML from 'yaml';
 import {asyncForEach, camelize, copyFile, normalize, readFile, writeFile} from './utils';
 
+export function detailNameBuilder(data, template): string {
+  // TODO: title isn't the best option, maybe original filename, or a new field?
+  return template.outputFile.replace(/%s/g, normalize((data.data.title)));
```
Comment (kotzendekrabbe) [inline src/renderer.ts:12]: would be title + location be an option? what do you think?

**Review by kotzendekrabbe — CHANGES_REQUESTED** [review]
only one idea for the url  :)

File: src/renderer.ts (hunk 1 above)
Comment (chaosmonster) [reply src/renderer.ts:12]: could work - would also help to not search for the same position with two different texts

Comment (chaosmonster) [reply src/renderer.ts]: done

**Pull request merged**